libs/vkd3d/resource.c

```diff
@@ -3000,7 +3002,9 @@ HRESULT d3d12_resource_create_placed(struct d3d12_device *device, const D3D12_RE
             &heap->desc.Properties, heap->desc.Flags, initial_state, optimized_clear_value, &object)))
         return hr;
 
-    object->heap = heap;
+    /* https://learn.microsoft.com/en-us/windows/win32/api/d3d12/nf-d3d12-id3d12device-createheap#remarks.
```

Comment:
Super nit, but the `.` at the end of the URL prevents the anchor from actually working (does not scroll to remarks)